assets/js/theme/common/faceted-search.js

```diff
@@ -380,10 +380,17 @@ class FacetedSearch {
             return;
         }
 
-        const url = Url.parse(window.location.href);
-        const queryParams = decodeURI($(event.currentTarget).serialize());
+        const url = Url.parse(window.location.href, true);
```

Comment:
We should also add a test for this change https://github.com/bigcommerce/cornerstone/blob/master/assets/js/test-unit/theme/common/faceted-search.spec.js